packages/hint-no-protocol-relative-urls/src/hint.ts

```diff
@@ -54,6 +54,17 @@ export default class NoProtocolRelativeUrlsHint implements IHint {
                 debug('Protocol relative URL found');
 
                 const message = getMessage('noProtocolRelativeUrl', context.language);
+                const attribute = src ? 'src' : 'href';
+                const attributeLocation = element.getAttributeLocation(attribute);
+                const fixedUrl = url.replace('//', 'https://');
+                const replacementText = `${attribute}="${fixedUrl}"`;
+
+                const codeFix = [
```

Comment:
@mliao95 want to take a stab at adding a test to the suite for this hint to verify the proposed fix matches what we expect? 

At a minimum we can just compare the returned location and text to hard-coded values, but even better if we can enhance our test helpers to apply the replacement so we can verify the resulting text matches our expectations instead.